File: php/liner_search.php
<?php
$arr=array(1,2,3,4,5,6,7);
$search_element=0;
$result=search($arr,$search_element);
if($result!=-1){
    echo "Element ".$search_element." found on index ".$result;
}else{
    echo "Element ".$search_element." not found";
}



function search($arr,$search_element){
   $element_on_index=-1;
  foreach($arr as $key=>$value){
      if($value==$search_element){
        $element_on_index=$key;
        break;
      }
  }
  return $element_on_index;
}

File: java/LinerSearch.java
class LinerSearch{  

    public static void search_element(int arr[],int search){
        
        boolean result=false;
         
        for(int i=0;i<arr.length;i++){
           
            if(search==arr[i]){
              result=true;
              break;
            }

        }

        if(result==true){
            System.out.println("Element "+search+" found in an array");
        }else{
            System.out.println("Element "+search+" not found in an array");
        }
        
    }

    public static void main(String args[]){  
     int array[]={1,2,3,4,5,6};  
     search_element(array,1);
    }  
}  